test/api/test_ingestiblec_json.py

```diff
@@ -46,65 +46,72 @@ def tearDown(self):
     """
     Purge all objects and collections created in the test case
     """
-    self.delete_test_article()
+    #self.delete_test_article()
 
   def test_ingestible(self):
     """
     GET ingestibles
     """
     print('\nTesting GET ingestibles/\n')
     # Invoke ingestibles API
-    self.get_ingestible(resources.ARTICLE_DOI)
+    self.get_ingestible(resources.RELATED_ARTICLE_DOI)
     self.verify_http_code_is(resources.OK)
+    self.delete_test_article(resources.RELATED_ARTICLE_DOI, resources.NOT_SCAPE_RELATED_ARTICLE_DOI,
+                             resources.RELATED_ARTICLE_BUCKET_NAME)
 
-  def delete_test_article(self):
+  def delete_test_article(self, article_doi, not_scaped_article_doi, bucket_name):
     """
     Gets article information for rhino then proceeds to delete article records from ambra db
     and content repo database
     :param None
     :return: None
     """
-    self.get_article(resources.ARTICLE_DOI)
-    if self.get_http_response().status_code == resources.OK:
-      # Deletes article from ambra database
-      self.delete_article_sql_doi(resources.NOT_SCAPE_ARTICLE_DOI)
-      #Delete CRepo collections
-      self.delete_test_collections()
-      #Delete CRepo objects
-      self.delete_test_objects()
-    else:
-      print (self.parsed.get_attribute('message'))
+    try:
+      self.get_article(article_doi)
+      if self.get_http_response().status_code == resources.OK:
+        self.delete_article_sql_doi(not_scaped_article_doi)
+        #Delete article
+        self.delete_article(article_doi)
+        self.verify_http_code_is(resources.OK)
+        #Delete CRepo collections
+        self.delete_test_collections(article_doi,bucket_name)
+        #Delete CRepo objects
+        self.delete_test_objects(article_doi,bucket_name)
+
+      else:
+        print (self.parsed.get_attribute('message'))
+    except:
```

Comment:
exceptions should not be generic, change it to a particular one, and should not pass silently (with "pass"), it should have some action or description.